src/patternfly/components/Nav/nav-link.hbs

```diff
@@ -13,12 +13,12 @@
   {{#if nav-item--IsFlyout}}
     aria-haspopup="true"
   {{/if}}
-  {{#ifAny nav-item--IsFlyout nav-item--IsDrilldown nav-item--IsExpandable nav-item--IsExpandable}}
-    {{#ifAny nav-item--expanded nav-item--IsExpanded}}
+  {{#ifAny nav-item--IsFlyout nav-item--IsDrilldown nav-item--IsExpandable}}
+    {{#if nav-item--IsExpanded}}
```

Comment:
Also, here you can use the `ternary` helper to dry out this block. That'll removing 4 lines of code. If we do this regularly, we can reduce `.hbs` partials file sizes significantly and eliminate the need for closing tags (`if/ifAny/etc`). It's also reduces complexity by improving scanability. Single quotes denote a string literal since the returned value is `'true/false'`.  

```
aria-expanded="{{ternary nav-item--IsExpanded 'true' 'false'}}"
```

vs 

```
{{#if nav-item--IsExpanded}}
  aria-expanded="true"
{{else}}
  aria-expanded="false"
{{/if}}
```